# English (United States) translations for PROJECT.
# Copyright (C) 2017 ORGANIZATION
# This file is distributed under the same license as the PROJECT project.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2017.
#
msgid ""
msgstr ""
"Project-Id-Version: PROJECT VERSION\n"
"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
"POT-Creation-Date: 2017-05-09 18:11+0800\n"
"PO-Revision-Date: 2017-04-14 15:38+0800\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language: en_US\n"
"Language-Team: en_US <LL@li.org>\n"
"Plural-Forms: nplurals=2; plural=(n != 1)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=utf-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 2.4.0\n"

#: app/apis/auth.py:58
msgid "注册成功"
msgstr ""

#: app/exceptions/auth.py:16
msgid "验证异常"
msgstr ""

#: app/exceptions/auth.py:25
msgid "需要登录"
msgstr ""

#: app/exceptions/auth.py:34
msgid "此邮箱未注册"
msgstr ""

#: app/exceptions/auth.py:43
msgid "密码错误"
msgstr ""

#: app/exceptions/auth.py:52
msgid "需要令牌"
msgstr ""

#: app/exceptions/auth.py:63
msgid "无效的令牌 ({message})"
msgstr ""

#: app/exceptions/base.py:20
msgid "未定义的错误"
msgstr ""

#: app/exceptions/v_code.py:16
msgid "验证码异常"
msgstr ""

#: app/exceptions/v_code.py:25
msgid "验证码已过期,请重新申请"
msgstr ""

#: app/exceptions/v_code.py:34
msgid "验证码错误"
msgstr ""

#: app/exceptions/v_code.py:43
msgid "验证码失效，请重新获取"
msgstr ""

#: app/exceptions/v_code.py:55
msgid "请等候{second}秒后再试"
msgstr ""

#: app/exceptions/v_code.py:57
msgid "冷却中,请稍后再试"
msgstr ""

#: app/models/user.py:114
msgid "这个用户还没有签名"
msgstr "This user has not signature"

#: app/validators/auth.py:20
msgid "此昵称已存在,请尝试其他昵称"
msgstr ""

#: app/validators/auth.py:27
msgid "此邮箱已存在,您可以直接登录或尝试其他邮箱"
msgstr ""

#: app/validators/auth.py:33 app/validators/auth.py:36
#: app/validators/auth.py:39 app/validators/auth.py:42
#: app/validators/auth.py:44 app/validators/auth.py:47
#: app/validators/auth.py:60 app/validators/auth.py:63
#: app/validators/auth.py:69
msgid "必填"
msgstr "Required"

#: app/validators/auth.py:33 app/validators/auth.py:60
#: app/validators/auth.py:69
msgid "邮箱格式不正确"
msgstr "Not a valid email address"

#: app/validators/auth.py:35 app/validators/auth.py:38
#: app/validators/auth.py:62
msgid "长度为{min}到{max}个字符"
msgstr "Length must be between {min} and {max}"

#~ msgid "此邮箱已存在,您可以直接登录"
#~ msgstr ""

#~ msgid "此昵称已存在"
#~ msgstr ""

#~ msgid "验证码异常基类"
#~ msgstr ""

